end_to_end_tests/golden-record/my_test_api_client/models/a_discriminated_union_type_1.py

```diff
@@ -1,38 +1,38 @@
-from typing import Any, Dict, List, Type, TypeVar, Union
+from typing import Any, Dict, List, Type, TypeVar
 
 from attrs import define as _attrs_define
 from attrs import field as _attrs_field
 
-from ..types import UNSET, Unset
-
 T = TypeVar("T", bound="ADiscriminatedUnionType1")
 
 
 @_attrs_define
 class ADiscriminatedUnionType1:
     """
     Attributes:
-        model_type (Union[Unset, str]):
```

Comment:
The changes in this generated file and the next one are because I changed `modelType` to be required in the test spec.